v2/cmd/wails/internal/commands/generate/template/base/main.tmpl.go

```diff
@@ -55,6 +55,7 @@ func main() {
 			DisableWindowIcon:    false,
 			// DisableFramelessWindowDecorations: false,
 			WebviewUserDataPath: "",
+			ZoomFactor: 1.0,
```

Comment:
I'll regenerate the templates to include this.

Reply:
Oh right, and for IsZoomControlEnabled :)
Thinking about it, I may add a note (and test) if one wants to apply a zoom but does allow runtime change by the users (f.e. using a config sets using a menu or other). That would be complete then, hopefully :)

Reply:
Added to the mdx :)